src/name/mlopatkin/andlogview/utils/SystemUtils.java

```diff
@@ -45,6 +45,11 @@
      */
     private static final String OS_NAME_WINDOWS_PREFIX = "Windows";
 
+    /**
+     * The prefix String for macOS.
+     */
+    private static final String OS_NAME_MACOS_PREFIX = "Mac OS X";
```

Comment:
Is it a copy-paste from the original file (commons lang3)?

Reply:
Nope, I didn't know about original file, just used the same approach as in `IS_OS_WINDOWS` below.